cmd/goal/ledger.go

```diff
@@ -22,8 +22,18 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	round       uint64
+	rawFilename string
+)
+
 func init() {
 	ledgerCmd.AddCommand(supplyCmd)
+	ledgerCmd.AddCommand(rawBlockCmd)
+
+	rawBlockCmd.Flags().Uint64VarP(&round, "round", "r", 0, "The round to fetch the raw block for")
+	rawBlockCmd.Flags().StringVarP(&rawFilename, "out", "o", stdoutFilenameValue, "The filename to dump the raw block to (if not set, use stdout)")
```

Comment:
Might be nicer UX to have this as an argument rather than a flag (e.g., `goal ledger rawblock 500` instead of `goal ledger rawblock -r=500`.)

Reply:
done